aci/resource_aci_vmmvswitchpolicycont.go

```diff
@@ -0,0 +1,433 @@
+		if err != nil {
+			return err
+		}
+
+	}
+
+	if relationTovmmRsVswitchOverrideStpPol, ok := d.GetOk("relation_vmm_rs_vswitch_override_stp_pol"); ok {
+		relationParam := relationTovmmRsVswitchOverrideStpPol.(string)
+		err = aciClient.CreateRelationvmmRsVswitchOverrideStpPol(vmmVSwitchPolicyCont.DistinguishedName, relationParam)
+		if err != nil {
+			return err
+		}
+
+	}
+
+	if relationTovmmRsVswitchOverrideLldpIfPol, ok := d.GetOk("relation_vmm_rs_vswitch_override_lldp_if_pol"); ok {
+		relationParam := relationTovmmRsVswitchOverrideLldpIfPol.(string)
+		err = aciClient.CreateRelationvmmRsVswitchOverrideLldpIfPol(vmmVSwitchPolicyCont.DistinguishedName, relationParam)
+		if err != nil {
+			return err
+		}
+
+	}
+
+	if relationTovmmRsVswitchOverrideMcpIfPol, ok := d.GetOk("relation_vmm_rs_vswitch_override_mcp_if_pol"); ok {
+		relationParam := relationTovmmRsVswitchOverrideMcpIfPol.(string)
+		err = aciClient.CreateRelationvmmRsVswitchOverrideMcpIfPol(vmmVSwitchPolicyCont.DistinguishedName, relationParam)
+		if err != nil {
+			return err
+		}
+
+	}
+
+	if relationTovmmRsVswitchOverrideCdpIfPol, ok := d.GetOk("relation_vmm_rs_vswitch_override_cdp_if_pol"); ok {
+		relationParam := relationTovmmRsVswitchOverrideCdpIfPol.(string)
+		err = aciClient.CreateRelationvmmRsVswitchOverrideCdpIfPol(vmmVSwitchPolicyCont.DistinguishedName, relationParam)
+		if err != nil {
+			return err
+		}
+
+	}
+
+	if relationTovmmRsVswitchOverrideLacpPol, ok := d.GetOk("relation_vmm_rs_vswitch_override_lacp_pol"); ok {
+		relationParam := relationTovmmRsVswitchOverrideLacpPol.(string)
+		err = aciClient.CreateRelationvmmRsVswitchOverrideLacpPol(vmmVSwitchPolicyCont.DistinguishedName, relationParam)
+		if err != nil {
+			return err
+		}
+
+	}
+
+	if relationTovmmRsVswitchOverrideMtuPol, ok := d.GetOk("relation_vmm_rs_vswitch_override_mtu_pol"); ok {
+		relationParam := relationTovmmRsVswitchOverrideMtuPol.(string)
+		err = aciClient.CreateRelationvmmRsVswitchOverrideMtuPol(vmmVSwitchPolicyCont.DistinguishedName, relationParam)
+		if err != nil {
+			return err
+		}
+
+	}
+
+	d.SetId(vmmVSwitchPolicyCont.DistinguishedName)
+	return resourceAciVSwitchPolicyGroupRead(d, m)
+}
+
+func resourceAciVSwitchPolicyGroupUpdate(d *schema.ResourceData, m interface{}) error {
+	aciClient := m.(*client.Client)
+	desc := d.Get("description").(string)
+
+	VMMDomainDn := d.Get("vmm_domain_dn").(string)
+
+	vmmVSwitchPolicyContAttr := models.VSwitchPolicyGroupAttributes{}
+	if Annotation, ok := d.GetOk("annotation"); ok {
+		vmmVSwitchPolicyContAttr.Annotation = Annotation.(string)
+	}
+	if NameAlias, ok := d.GetOk("name_alias"); ok {
+		vmmVSwitchPolicyContAttr.NameAlias = NameAlias.(string)
+	}
+	vmmVSwitchPolicyCont := models.NewVSwitchPolicyGroup(fmt.Sprintf("vswitchpolcont"), VMMDomainDn, desc, vmmVSwitchPolicyContAttr)
+
+	vmmVSwitchPolicyCont.Status = "modified"
+
+	err := aciClient.Save(vmmVSwitchPolicyCont)
+
+	if err != nil {
+		return err
+	}
+
```

Comment:
There should one pre check for all relations. That will check whether relation DN actually exist or not.